@xen-orchestra/web/src/pages/host/[id]/network.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="host-network-view">
+    <div class="host-network-table-container">
+      <UiTitle type="h4" class="header">
+        <slot>{{ $t('pifs') }}</slot>
+        <template #actions>
+          <UiButton size="medium" variant="secondary" accent="info" :left-icon="faArrowsRotate">
+            {{ $t('scan-pifs') }}
+          </UiButton>
+        </template>
+      </UiTitle>
+      <div v-if="isReady" class="table-container">
+        <UiQuerySearchBar class="table-query" @search="(value: string) => (filter = value)" />
+        <UiTableActions>
+          <UiButton size="medium" variant="tertiary" accent="info" :left-icon="faEdit" disabled>
+            {{ $t('edit') }}
+          </UiButton>
+          <UiButton size="medium" variant="tertiary" accent="info" :left-icon="faTrash" disabled>
+            {{ $t('delete') }}
+          </UiButton>
+          <template #title>
+            <UiActionsTitle> {{ $t('table-actions') }}</UiActionsTitle>
+          </template>
+        </UiTableActions>
+        <VtsTable vertical-border class="table">
+          <thead>
+            <tr>
+              <ColumnTitle id="checkbox">
+                <UiCheckbox accent="info" />
+              </ColumnTitle>
+              <ColumnTitle id="network" :icon="faAlignLeft">{{ $t('network') }}</ColumnTitle>
+              <ColumnTitle id="device" :icon="faAlignLeft">{{ $t('device') }}</ColumnTitle>
+              <ColumnTitle id="status" :icon="faPowerOff">{{ $t('status') }}</ColumnTitle>
+              <ColumnTitle id="vlan" :icon="faAlignLeft">{{ $t('vlan') }}</ColumnTitle>
+              <ColumnTitle id="ipAddresses" :icon="faAt">{{ $t('ip-addresses') }}</ColumnTitle>
+              <ColumnTitle id="macAddress" :icon="faAt">{{ $t('mac-address') }}</ColumnTitle>
+              <ColumnTitle id="ipMode" :icon="faCaretDown">{{ $t('ip-mode') }}</ColumnTitle>
+              <ColumnTitle id="more">
+                <UiButtonIcon :icon="faEllipsis" size="medium" accent="info" />
+              </ColumnTitle>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="pif in pifs" :key="pif.id">
+              <VtsCellText>
+                <UiCheckbox accent="info" />
+              </VtsCellText>
+              <VtsCellText>
+                <UiObjectLink>
+                  <template #icon>
+                    <UiComplexIcon size="medium">
+                      <VtsIcon :icon="faNetworkWired" accent="current" />
+                      <VtsIcon accent="success" :icon="faCircle" :overlay-icon="faCheck" />
+                    </UiComplexIcon>
+                  </template>
+                  {{ getNetworkName(pif) }}
+                </UiObjectLink>
+              </VtsCellText>
+              <VtsCellText>{{ pif.device }}</VtsCellText>
+              <VtsCellObject class="pif-status">
+                <PifStatus :icon="faCircle" :pif="pif" />
```

Comment:
You can use the shorthand here:
```suggestion
                <PifStatus :icon="faCircle" :pif />
```